README.md

```diff
@@ -16,7 +16,6 @@ By the end of this workshop you will understand and have experience with the fol
 * Zero code CRUD using data binding
 * Rendering repeated items using repeaters
 * Wix data hooks
-* Wix data events (not to be confused with the wix events vertical)
```

Comment:
I meant the edm events as you said in the beginning :)

Reply:
Ah oops misunderstanding.. [wdyt about this](https://github.com/ofirdagan/velo-workshop/pull/1/commits/5d0010847ba957a8d355c607fdd944d15c0866cd)?